src/main/interface/settings.c

```diff
@@ -461,6 +469,10 @@ const lookupTableEntry_t lookupTables[] = {
     LOOKUP_TABLE_ENTRY(lookupTableRcSmoothingInputType),
     LOOKUP_TABLE_ENTRY(lookupTableRcSmoothingDerivativeType),
 #endif // USE_RC_SMOOTHING_FILTER
+#ifdef USE_GYRO_DATA_ANALYSE
+    LOOKUP_TABLE_ENTRY(lookupTableDynamicFilterLocation),
```

Comment:
Looking good. Now I'd go for a

```
enum {
	DYNAMIC_FILTER_POS_PRE = 0,
	DYNAMIC_FILTER_POS_POST,
};
```

in `gyro.c`, and then use this for the checks (also, n.b. the values in lookups are 0 based).

Reply:
OK I tried to make an enum, but now the code won't compile - rank beginner am I...

Reply:
```
./src/main/sensors/gyroanalyse.c: In function 'gyroDataAnalyseInit':
./src/main/sensors/gyroanalyse.c:116:46: error: 'DYNAMIC_FILTER_POS_PRE' undeclared (first use in this function)
     if (gyroConfig()->dyn_filter_location == DYNAMIC_FILTER_POS_PRE) {
```

Reply:
Sorry I give up on this one, if you could fix it, or show me how, I'll happily make the changes.

Reply:
Maybe fixed it?  Put it into gyro.h:
```
typedef enum {
    DYNAMIC_FILTER_POS_PRE = 0,
    DYNAMIC_FILTER_POS_POST
} dyn_filter_location_e;
```